doc/prow/4.7/query_upgrade_sanity.json

```diff
@@ -0,0 +1,3 @@
+{
+  "case_query": "products.KEY:ocp AND NOT status:inactive AND type:testcase AND caseautomation.KEY:automated AND automation_script:cucushift AND NOT upstream.KEY:yes AND env_install_method.KEY:(__all__ ipi) AND caseimportance.KEY:critical AND version.KEY:4_7 AND NOT tags:(prod_only stage_only)"
```

Comment:
Perhaps reflect ipi in the file name ?

Reply:
Thanks, I'll correct the query and test cases, the query should include critical tests, regardless of installation methods.

Reply:
Updated, the query result did not change, so no new tags added